tests/coverage/mcdc_non_control_flow.rs

```diff
@@ -4,6 +4,9 @@
 //@ compile-flags: -Zcoverage-options=mcdc
 //@ llvm-cov-flags: --show-mcdc
 
+// This test ensures that boolean expressions that are not inside control flow
+// decisions are correctly instrumented.
+
```

Comment:
Nitpick: Ideally these changes should be in the patch that adds the original test, so that the re-bless patch doesn't have irrelevant line number changes.

Reply:
Will do :+1: